dataset_config/template/deploy.json

```diff
@@ -5,11 +5,14 @@
 // The project in this file may be different from the project containing
 // dataset BigQuery tables.
 // 
-// es_java_opts:
-//   Optional. Set if you need more than default Elasticsearch heap size of 1G.
-//   This is only used for GKE deployment. For local deployment, use:
-//     ES_JAVA_OPTS="-Xms10g -Xmx10g" docker-compose up -d elasticsearch
+// data_node_machine_type:
```

Comment:
data_node_machine_type -> node_pool_machine_type
data_node_num_repliacs -> node_pool_num_nodes
n1-standard-1 -> n1-standard-4

Same for rest of PR (including README).

Also change 1000 Genomes deploy.json. That way, someone can copy the 1000 Genomes files and they'll have all the flags.

Reply:
Done.